Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN bundle --without test development
 COPY . .
 
 RUN ln -sf /opt/conjur-server/bin/conjurctl /usr/local/bin/
+RUN ln -sf /opt/conjur-server/engines/conjur_audit/db/migrate/* /opt/conjur-server/db/migrate
```

Comment:
We were trying to use native Engine tasks, however, since we are using `Sequel` gem that is replacing `activerecord` following suggested task: 
`rake conjur_audit:install:migrations`

Chaining this activerecord tasks:
https://github.com/rails/rails/blob/212481b490bcd5db77aae1a0a338e25eedc98471/activerecord/lib/active_record/railties/databases.rake#L357

**Is not implemented in `Sequel-Rails`**
https://github.com/TalentBox/sequel-rails/blob/master/lib/sequel_rails/railties/database.rake

Thus can't be use to copy files. 
Any other solutions without enabling activerecord (which we are not using)?